src/elements/content-picker/ContentPicker.js

```diff
@@ -1235,7 +1235,7 @@ class ContentPicker extends Component<Props, State> {
                         >
                             <Pagination
                                 offset={offset}
-                                onChange={this.paginate}
+                                onOffsetChange={this.paginate}
```

Comment:
Originally, the `Pagination` component only handled offset-based paging, which is largely incompatible with marker-based paging. Why were they merged into a single component? Perhaps just use the OffsetBasedPagination component directly?

Reply:
@jstoffan They are not merged. In fact, the earlier `<Pagination />` component only supported offset-based pagination. As part of https://github.com/box/box-ui-elements/pull/1854, we added ability for marker based pagination which resulted in `< OffsetBasedPagination  />` and `<MarkerBasedPagination />`. `<Pagination />` was just refactored to be a simple wrapper around these.

Reply:
@manishPh @jstoffan I updated this PR to use `OffsetBasedPagination` instead of `Pagination`. Could you review again?